README.rst

```diff
@@ -1,5 +1,10 @@
-rst2pdf: Use a text editor. Make a PDF.
-========================================
+=======
+rst2pdf
+=======
+
+------------------------------
+Use a text editor. Make a PDF.
+------------------------------
```

Comment:
Moving all the heading markers down a level will affect https://pypi.org/project/rst2pdf/. Are we okay with that?

Reply:
I think it shouldn't matter. You can choose whatever symbols you like, in any order, so long as each level has a unique character.